### PR TITLE
Issue hash and signature URLs of advisories in ROLIE feeds, too. 

### PR DESCRIPTION
**Pull request by s-l-teichmann**
For #74.

### CODE REVIEW

File: cmd/csaf_aggregator/indices.go [hunk 1]
```diff
@@ -166,10 +166,11 @@ func (w *worker) writeROLIE(label string, summaries []summary) error {
 			Titel:     s.summary.Title,
 			Published: csaf.TimeStamp(s.summary.InitialReleaseDate),
 			Updated:   csaf.TimeStamp(s.summary.CurrentReleaseDate),
-			Link: []csaf.Link{{
-				Rel:  "self",
-				HRef: csafURL,
-			}},
+			Link: []csaf.Link{
+				{Rel: "self", HRef: csafURL},
+				{Rel: "hash", HRef: csafURL + ".sha512"},
```
Comment (tschmidtb51) [inline cmd/csaf_aggregator/indices.go:171]: As we compute the `sha256` as well, is there a reason for not adding it?

Comment (s-l-teichmann) [reply cmd/csaf_aggregator/indices.go:171]: No particular reason ... I just saw the .sha512 in the example linked in #74.
We can add .sha256 as well. Also as rel "hash"?

Comment (tschmidtb51) [reply cmd/csaf_aggregator/indices.go:171]: Yes.

Comment (s-l-teichmann) [reply cmd/csaf_aggregator/indices.go:172]: [x] done

Comment (tschmidtb51) [reply cmd/csaf_aggregator/indices.go:172]: Perfect.

**Pull request merged**